sql/migrations/042_jira_athenian_issues.up.sql

```diff
@@ -0,0 +1,51 @@
+BEGIN;
+
+CREATE TABLE jira.athenian_issue (
+    acc_id bigint NOT NULL,
+    id text NOT NULL,
+    work_began timestamptz,
+    resolved timestamptz,
+    PRIMARY KEY(acc_id, id),
+    CONSTRAINT fk_acc_id
+        FOREIGN KEY(acc_id)
+            REFERENCES jira.installation(acc_id)
+                ON DELETE CASCADE
+);
+
+CREATE OR REPLACE FUNCTION jira.get_transition_timestamp (account_id bigint, cur_issue_id text, status_category_key text)
+    RETURNS timestamptz AS $timestamp$
+declare
+    timestamp timestamptz;
+BEGIN
+    select created into timestamp from jira.issue_changelog
+    where acc_id = $1 and issue_id = $2 and lower(field) = 'status'
+      and to_ids[1] in (select id from jira.status where acc_id = $1
+                                                     and status_category_id in(select id from jira.status_category where acc_id = $1 and key = $3)) order by created limit 1;
+    RETURN timestamp;
+END;
+$timestamp$ LANGUAGE plpgsql;
```

Comment:
```suggestion
$$ LANGUAGE plpgsql;
```
same in other places

Reply:
Done.